src/frontend/src/components/SearchPage/SearchPage.jsx

```diff
@@ -76,7 +77,14 @@ const SearchPage = () => {
             },
           };
         });
-      setResults({ type: 'author', searchResults: finalResults });
+      const timeline = data.getPosts.map((result) => {
+        return {
+          id: result.id,
+          url: `/posts/${result.id}`,
+          author: result.feed.author,
```

Comment:
Could this be a possible issue if we have two feed `authors` with the exact name?

Reply:
ahh true i didn't think of that. Maybe we could use the feed id instead then?